src/app/components/MessageBanner/index.test.tsx

```diff
@@ -109,5 +110,44 @@ describe('MessageBanner', () => {
       const image = screen.getByAltText('');
       expect(image).toBeInTheDocument();
     });
+    describe('Presence on live environment', () => {
+      const originalEnvironment = process.env.SIMORGH_APP_ENV;
+
+      afterEach(() => {
+        process.env.SIMORGH_APP_ENV = originalEnvironment;
+      });
+      it('should not render banner when the environment is live', () => {
```

Comment:
Is this test going to be redundant when the `MessageBanner` is live and on a page?

Reply:
Yep, the isLive function in MessageBanner will be deleted when the banner goes live which should make this test fail and therefore no longer needed!